starter-code/routes/index.js

```diff
@@ -1,9 +1,63 @@
-const express = require('express');
-const router  = express.Router();
+const express = require("express");
+const router = express.Router();
+const Celebrity = require("../models/Celebrity.js");
 
 /* GET home page */
-router.get('/', (req, res, next) => {
-  res.render('index');
+router.get("/", (req, res, next) => {
+  res.render("index");
+});
+
+// ****************
+
```

Comment:
Adding all your celebrity routes here is ok for now, but for your own project I recommend organising them by topic because it will make your code a lot easier to follow